consumers/client-event-consumer/src/main/java/com/flipkart/aesop/event/AbstractEventFactory.java

```diff
@@ -118,4 +100,40 @@ public AbstractEvent createEvent(Map<String, Object> fieldsMap, Set<String> prim
 
 		return event;
 	}
+
+	/**
+	 * This returns mysql object from avro object and intented mysql datatype
+	 * @param mysqlType
+	 * @param fieldValue
+	 * @return MysqlTypedObject using AvroToMysqlMapper
+	 */
+	private Object getMysqlTypedObject(String mysqlType, Object fieldValue)
+	{
+		MysqlDataTypes mysqlTypeToConvert = MysqlDataTypes.valueOf(mysqlType.toUpperCase());
+		Object mysqlTypedObject = AvroToMysqlMapper.avroToMysqlType(fieldValue, mysqlTypeToConvert);
```

Comment:
Also, can you double check the convention used for mySql in this codebase? Is it mySql or mysql